### DISCUSSION TITLE
QCG output bug?

### DISCUSSION THREAD

**Discussion by alan-arnold** [grimme-lab/xtb#727] [Q&A] [answered]
Hi. I've just starting doing some explorations with QCG (xtb 6.5.1 / crest 2.12) like this one:

crest test.xyz --T 24 --qcg h2o.xyz --gsolv --nsolv 50 --alpb h2o --nofix --mdlen 10 -prop hess -keepdir > test.out &

and notice that the ref. states in the headings near the end of the output file are identical, but the results are not.   

  Gsolv and Hsolv ref. state: [**1 M gas/solution**]                <<< Why is this the same as below?
  G_solv (incl.RRHO)      =  -81.17 kcal/mol
  H_solv (incl.RRHO)      =  -90.04 kcal/mol

  Gsolv and Hsolv ref. state: [**1 M gas/solution**]                
  G_solv (incl.RRHO)      =  -83.06 kcal/mol
  H_solv (incl.RRHO)      =  -91.93 kcal/mol

btw, is the -prop hess option redundant in a QCG gsolv run?

**Comment by cplett** [comment, marked as answer]
Hi,
You are right that there is a mistake in the output. The upper one should not be printed (these are the values without the volume work). Thank you for pointing this out. 
The `-prop hess` option has currently no effect on the QCG run as the solvation free energies are anyhow evaluated in terms of free energies.

**Reply by alan-arnold** [reply]
Thanks Christoph for your quick reply.
